src/irmin/content_addressable_intf.ml

```diff
@@ -32,22 +29,19 @@ module type S = sig
       content-addressable store to generate a consistent key. *)
 
   val unsafe_add : [> write ] t -> key -> value -> unit Lwt.t
```

Comment:
I think we should stick to `key` in the comments, instead of `hash`. (its a bit of a shame that `key=hash` is not reflected in the type for CA stores)

Reply:
Sure. I think the comment change comes from an intermediate state in which `Content_addressable.S` was included elsewhere (in which case having inlined documentation refer to two different "key"s was confusing), but that's not the case at the moment so I'll revert it.